test/dlc_tests/ec2/mxnet/training/test_mxnet_training.py

```diff
@@ -58,6 +58,7 @@ def test_mxnet_keras_cpu(mxnet_training, ec2_connection, cpu_only):
     execute_ec2_training_test(ec2_connection, mxnet_training, MX_KERAS_CMD)
 
 
+@pytest.mark.skip(reason="Skip until DGL with cuda 11.0 is available")
```

Comment:
Please insert this skip inside the test function and check for `cu110` in the image tag, rather than just skipping for all GPU images, so that older framework versions can still be tested.

Reply:
Moved skips to inside functions, only if cu110 present in image tag.